web/src/components/pages/contribution/speak/speak.tsx

```diff
@@ -55,37 +63,38 @@ const UnsupportedInfo = () => (
   <div className="empty-container">
     <div className="error-card card-dimensions unsupported">
       <Localized id="record-platform-not-supported" />
-      <p key="desktop">
+      <p className="desktop" key="desktop">
         <Localized id="record-platform-not-supported-desktop">
           <span />
         </Localized>
-        <a target="_blank" href="https://www.firefox.com/">
-          <FontIcon type="firefox" />
-          Firefox
+      </p>
+      <div>
+        <a target="_blank" href="https://www.firefox.com/" title="Firefox">
```

Comment:
We should add `rel="noopener noreferrer"` on these links. I just grepped our front-end and realized that a few of our `target="_blank"` links are also missing those attributes, which we should definitely fix.